ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -539,6 +334,13 @@ public final class AliasTokens: NSObject {
         case brandStroke1Pressed
         case brandStroke1Selected
     }
+
+    @available(swift, obsoleted: 1.0, message: "This method exists for Objective-C backwards compatibility and should not be invoked from Swift. Please use the `brandColors` property directly.")
+    @objc(aliasColorForToken:)
```

Comment:
For consistency this should be aliasColorsForToken (plural, like the the colors token set below)

Reply:
It seems the consistent behavior among existing tokens was to name it like this
<img width="801" alt="Screenshot 2023-01-03 at 6 17 14 PM" src="https://user-images.githubusercontent.com/106181067/210457200-99ad6f40-d116-4082-9493-62e5053e0923.png">

Reply:
singular makes more sense because the the function returns a color.